Ida.adjivon/docs 10781 pana edits (#29131)
DataDog/documentation: content/en/product_analytics/guide/rum_and_product_analytics.md

## FAQs

### As part of the introduction of Product Analytics, the following changes are being made for both existing and new RUM customers:

As part of the introduction of Product Analytics, the following changes are being made for both existing and new RUM customers:

- **Product Analytics Summary**, **Retention Analytics**, and **Pathways** are now part of **Product Analytics** and are no longer available within RUM.

- **Heatmaps** remain available but have been relocated to the **Session Replay** tab for easier access and contextual relevance.

- The **Funnel** and **Conversion** tabs continue to be available in the **RUM Explorer**. To view conversion details, select the **Funnel** visualization and click on any step to open the **Conversion** panel.

These updates ensure a clearer separation of use cases between RUM and Product Analytics, enabling more focused workflows and deeper insights.

### Why can I no longer access Sankeys and Retention Analysis?
- These features were part of a preview and available to RUM customers as a beta, but were not part of the core RUM offering. These features are exclusively available to customers on a Product Analytics contract.

> ### Setting up Product Analytics is simple and uses the same SDKs and configuration as Real User Monitoring (RUM)
> 
> - **Shared Setup**: Product Analytics and RUM collect data using the same SDKs and require identical implementation steps.
> 
> - **Enablement**: To enable RUM, Product Analytics, or both, go to your **Application Detail** page and toggle the desired products.
> 
> No additional instrumentation is required if RUM is already configured.

Reviewer: ```suggestion
### How do I set up Product Analytics?

Product Analytics uses the same SDKs and configuration as RUM. To enable Product Analytics...

If RUM is already configured, no additional instrumentation is required for Product Analytics.
```

rewrote to phrase the heading as a question, and to make things a bit clearer... but what's the Application Detail page? I thought things happened in the Plan & Usage page, but I could be totally wrong

I also feel like this question has some pretty important information and should maybe be moved up the page, maybe right after the first question?

Author: It's now called the [Application Management](https://app.datadoghq.com/rum/list?fromUser=false&refresh_mode=sliding&from_ts=1746140463512&to_ts=1746226863512&live=true) page it seems. 

made that edit and added the link
